solr/docker/tests/cases/version/test.sh

```diff
@@ -1,45 +0,0 @@
-#!/bin/bash
```

Comment:
Deleted because we can set the tag and version to be anything now?

Reply:
Yeah, this is something that might be useful when we start creating release artifacts, but the test we want will probably look quite different. This is pretty shallow and doesn't actually provide a whole lot.